index.html

```diff
@@ -199,7 +199,7 @@ <h5 class="font-weight-medium mb-4">S.O.L.I.D principles</h5>
             </div>
             <div class="d-flex justify-content-center mb-3">
```

Comment:
You need to change this attributes, not the footer.

```
    <meta content="Free HTML Templates" name="keywords">
    <meta content="Free HTML Templates" name="description">
```

Important, you must check the lincense of the assets that you are using, for example the HTML template that you are using requires you to maintain the `Designed by <a class="text-white font-weight-bold" href="https://htmlcodex.com">HTML Codex</a>`